src/cloud/release-notes/ece-release-notes.md

```diff
@@ -17,9 +17,26 @@ The `{{site.data.var.ct}}` package uses the following release versioning sequenc
 {:.bs-callout-info}
 See [Upgrades and patches]({{ site.baseurl }}/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `{{site.data.var.ct}}` package.
 
-## v2002.1.3
+## v2002.1.4
 *Release date: {{ site.data.var.ece-release-date }}*<br/>
 
+-  {:.fix}Fixed an issue that caused deployment failure when the search engine specified in the `SEARCH_CONFIGURATION` environment variable is a value other than `elasticsearch`.<!--MCLOUD-7283-->
+
+-  {:.fix}Updated the database service validation to support Magento projects configured with a [custom database connection]({{ site.baseurl }}/cloud/env/variables-deploy.html#database_configuration) to the Amazon Aurora MySQL database. This fix resolves the following types of deploy errors that can occur on projects with custom connections to Aurora:<!--MCLOUD-7269-->
+
+   ```terminal
+   Version of service `mysql` is not detected
+   ```
+
+   ```terminal
+   The current configuration is not compatible with this version of Magento.
+   Magento 2.4.1.0 does not support version "5.7" for service "mysql".
+   Service version should satisfy ">=10.2 <10.5" constraint.
+   ```
```

Comment:
This shouldn't be in the current release notes as work with Aurora is not finished yet, and PR is not merged in ece-tools

Reply:
In `v2002.1.4` we only fixed the issue with `SEARCH_CONFIGURATION`

Reply:
@oshmyheliuk  and @NadiyaS Removed that release note and associated doc update. Thanks for the clarification.